src/mentorship/ml/models/hyperparams/tuning.py

```diff
@@ -0,0 +1,68 @@
+from sklearn.base import clone
```

Comment:
I think this module is specific to your current task, so let's move it to `kaggle/storesales` sub-folder.